lisa/operating_system.py

```diff
@@ -705,6 +706,7 @@ def add_repository(self, repo: str, key_location: str = "") -> None:
             expected_exit_code=0,
             expected_exit_code_failure_message="fail to add repository",
         )
+        self._initialize_package_installation()
```

Comment:
I remember the apt will update repos after add.

Reply:
tested and removed